test/jdk/java/util/TimeZone/DefaultTimeZoneTest.java

```diff
@@ -23,86 +23,94 @@
-                    button.setLabel(!english ? "English" : "Local");
-                }
-            });
-        button.setAlignmentX(Component.CENTER_ALIGNMENT);
-        JPanel panel = new JPanel();
-        panel.setLayout(new BoxLayout(panel, BoxLayout.PAGE_AXIS));
-        panel.add(Box.createRigidArea(new Dimension(0, 10)));
-        panel.add(tzid);
-        panel.add(Box.createRigidArea(new Dimension(0, 5)));
-        panel.add(label);
-        panel.add(Box.createRigidArea(new Dimension(0, 10)));
-        panel.add(button);
-        getContentPane().add(panel);
-    }
+    private static final SimpleDateFormat SDF =
+            new SimpleDateFormat("yyyy-MM-dd HH:mm:ss zzzz (XXX)");
+    private static final String INSTRUCTIONS =
+            """
+            Tests the platform time zone detection on all platforms.
+            (Part I) and on/off of DST adjustment on Windows (Part II).
 
-    @Override
-    public void start() {
-        clock = new Thread(this);
-        clock.start();
-    }
+            Part I:
+            Observe the displayed Time zone ID and the local time.
+            Change the platform time zone setting, then click the
+            "Update Time Zone" button. If the ID and local time
+            update correctly, part I passes, otherwise press fail. Note that
+            some time zone IDs have their aliases that may be displayed.
+            For example, "US/Pacific" is an alias of "America/Los_Angeles".
+            If this platform is Windows, proceed to Part II. Otherwise, press
+            the Pass button to complete this test.
 
-    @Override
-    public void stop() {
-        clock = null;
-    }
+            Part II:
+            Note that Part II may require the Administrator privilege to change
+            Windows setting.
 
-    @Override
-    public void run() {
-        Thread me = Thread.currentThread();
+              1. Open the Settings app and navigate to Time & Language > Date & Time
+              2. Select any time zone where daylight saving time is *currently*
+                 in effect, such as "(GMT-08:00) Pacific Time (US & Canada);
+                 Tijuana", "(GMT+10:00) Canberra, Melbourne, Sydney", and Apply.
+              3. After pressing "Update Time Zone" button, observe that the local
+                 time on the Settings app and the test local time are the same (daylight time).
+              4. Turn off "Adjust for daylight saving time automatically"
+              5. Observe the two local times should be the same (standard time).
+              6. Turn on "Adjust for daylight saving time automatically"
 
-        while (clock == me) {
-            // Reset the default time zone so that
-            // TimeZone.getDefault will detect the platform time zone
-            TimeZone.setDefault(null);
-            System.setProperty("user.timezone", "");
+            If the local time in the control panel and test window are always the same,
```

Comment:
nit: "control panel" -> "Settings app"

Reply:
FWIW on my WIndows 11 system both Settings and Control Panel provide this.

Reply:
Yes, but they are sunsetting Control Panel.